metals/src/main/scala/scala/meta/metals/MetalsServices.scala

```diff
@@ -480,6 +500,8 @@ class MetalsServices(
         fileSystemSemanticdbSubscriber.onNext(path)
       case CompilerConfig.File() =>
         compilerConfigSubscriber.onNext(path)
+      case _ if path == SbtServer.activeJson(cwd) =>
+        connectToSbtServer()
```

Comment:
Will this report an error on failure?

Reply:
Yes. This fires only if `active.json` was created or changed, so there should be running sbt server. And if we fail to connect to it, this failure will be reported.